hack/update-codegen.sh

```diff
@@ -128,6 +130,21 @@ ${informergen} \
   --go-header-file ${KUBE_ROOT}/hack/boilerplate/boilerplate.generatego.txt \
   "$@"
 
+registergen_external_apis=(
+$(
+  cd ${KUBE_ROOT}/staging/src
+  find k8s.io/api -name types.go | xargs -n1 dirname | sort
+)
+)
+registergen_external_apis_csv=$(IFS=,; echo "${registergen_external_apis[*]}")
+${registergen} \
```

Comment:
@sttts  when I run `register-gen` from this script the import tracker doesn't import `k8s.io/apimachinery/pkg/runtime` and `k8s.io/apimachinery/pkg/runtime/schema` but they are imported when I run it manually. Do you know how to fix it ?

Reply:
This is fixed?

Reply:
yes, see my previous comment https://github.com/kubernetes/kubernetes/pull/66839#discussion_r210272300